rest.go

```diff
@@ -89,9 +89,23 @@ func makeRESTAuth(authToken string) string {
 	return fmt.Sprintf("ModeCloud %s", authToken)
 }
 
-func makeRESTCall(method string, path string, authToken string, data interface{}, resData interface{}) error {
+func makeRESTCall(method string, path string, dc *DeviceContext, data interface{}, resData interface{}) error {
 	url := makeRESTCallURL(path)
 
+	authToken := ""
+	if dc != nil {
+		authToken = dc.AuthToken
```

Comment:
No need to set `authToken` if TLSClientAuth is used.

Reply:
fixed.